c10/cuda/test/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+# ---[ Test binaries.
+
+file(GLOB_RECURSE C10_CUDA_ALL_TEST_FILES *.cpp)
+if (BUILD_TEST)
+  foreach(test_src ${C10_CUDA_ALL_TEST_FILES})
+    get_filename_component(test_file_name ${test_src} NAME_WE)
+    set(test_name "c10_cuda_${test_file_name}")
+    add_executable(${test_name} "${test_src}")
+    target_link_libraries(${test_name} c10_cuda gtest_main)
+    add_test(NAME ${test_name} COMMAND $<TARGET_FILE:${test_name}>)
```

Comment:
You're creating new executables here. Please make sure they're actually run in oss CI